bodhi/server/util.py

```diff
@@ -604,7 +604,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
     if err:
-        log.error(err)
+        if p.returncode == 0:
+            log.info(err)
+        else:
+            log.error(err)
```

Comment:
Hey @bowlofeggs, thank you so much for the advice! You are so very helpful. Can't believe I didn't think of also mocking the logger... doh! :)

How do my tests look? I'm not sure whether it's sufficient to only check whether the output is being logged at stderr, or if I should also test whether they are being logged at info. Also, I'm a little inconsistent by mocking using the `with` statement *and* the decorator, but when I tried to use the following decorator: `@mock.patch('bodhi.server.log.error')`, I got the following error: `ValueError: need more than 0 values to unpack`. Not sure what that's about...but mocking using the `with` statement works flawlessly.